packages/runtime/container-runtime/src/dataStoreContext.ts

```diff
@@ -430,6 +430,18 @@ export abstract class FluidDataStoreContext extends TypedEventEmitter<IFluidData
         return undefined;
     }
 
+    /**
+     * After GC has run, called to notify this data store of routes that are used in it.
+     * @param usedRoutes - The routes that are used in this data store.
+     */
+    public updateUsedRoutes(usedRoutes: string[]) {
+        // Currently, only data stores can be collected. Once we have GC at DDS layer, the DDS' in the data store will
+        // also be notified of their used routes. See - https://github.com/microsoft/FluidFramework/issues/4611
+
+        // Update the used state of this data store in the summarizer node.
+        this.summarizerNode.used = usedRoutes.includes("/") || usedRoutes.includes("") ? true : false;
```

Comment:
nit: the ternary is unnecessary I think.
```suggestion
        this.summarizerNode.used = usedRoutes.includes("/") || usedRoutes.includes("");
```